### PR TITLE
Create Partnership Module

### PR DESCRIPTION
**Pull request by NextGestion**
Create Partnership Module

### CODE REVIEW

**Comment by NextGestion** [comment]
Fix Errors

**Comment by eldy** [comment]
Can you removed all files that we should not have into core and files with bad name like:
- files with a name with "--" into the name
- files with "object_" into name
- hidden files (.gitattributes, .editoconfig, .gitignore), we already have them into root of core.

**Comment by eldy** [comment]
There is errors reported by stickler-ci (Click on "details" to see them).
For example, you must not use 
`var $var`
into class
but 
`public $var`
Thanks to fix

**Comment by NextGestion** [comment]
Yes, we use "public $var".
**"stickler-ci"** detect a comment :

/**
	 * **@var string** ID to identify managed object.
	 */

**Comment by NextGestion** [comment]
> There is errors reported by stickler-ci (Click on "details" to see them).
> For example, you must not use
> `var $var`
> into class
> but
> `public $var`
> Thanks to fix

Yes, we use "public $var".
"stickler-ci" detect a comment :

/**
* @var string ID to identify managed object.
*/

**Comment by eldy** [comment]
> > There is errors reported by stickler-ci (Click on "details" to see them).
> > For example, you must not use
> > `var $var`
> > into class
> > but
> > `public $var`
> > Thanks to fix
> 
> Yes, we use "public $var".
> "stickler-ci" detect a comment :
> 
> /**
> 
> * @var string ID to identify managed object.
>   */

I'm afraid sticker has right.
I still has a lot of 
var $xxx
into code.
For example into class PartnershipUtils

Note that a PR can't be merged as long as there is error on CI checks.

File: htdocs/partnership/class/partnershiputils.class.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+require_once DOL_DOCUMENT_ROOT.'/core/lib/files.lib.php';
+dol_include_once('partnership/lib/partnership.lib.php');
+
+
+/**
+ *	Class with cron tasks of Partnership module
+ */
+class PartnershipUtils
+{
+	var $db;							//!< To store db handler
```
Comment (eldy) [inline htdocs/partnership/class/partnershiputils.class.php:37]: Wrong declaration

Comment (NextGestion) [reply htdocs/partnership/class/partnershiputils.class.php]: Ok we fixed it.

**Pull request merged**